dotnet/Liberator.Driver/RatDriverMethods.cs

```diff
@@ -364,6 +364,61 @@ public void ClickLinkAndWait(By locator, [Optional, DefaultParameterValue(true)]
             }
         }
 
+        /// <summary>
+        /// Clicks on a link and waits for a new page to be loaded that contains a specified URL or part URL
+        /// </summary>
+        /// <param name="element">The element on which to click</param>
+        /// <param name="url">The partial URL to be waited for</param>
+        /// <param name="waitForTarget">(Optional parameter) Whether to wait for the cliackability of the target element</param>
+        public void ClickLinkAndWaitForUrl(IWebElement element, string url, [Optional, DefaultParameterValue(true)] bool waitForTarget)
+        {
+            Element = element;
+            try
+            {
+                LastPage = Driver.FindElement(By.TagName("html"));
+                if (waitForTarget) { WaitForElementToBeClickable(element); }
+                if (typeof(TWebDriver) == typeof(OperaDriver) || typeof(TWebDriver) == typeof(InternetExplorerDriver))
+                {
+                    Element.SendKeys(Keys.Enter);
+                    WaitForUrlToContain(url);
+                }
+                else
+                {
+                    //TODO: IE & Opera currently not reporting staleness. To be investigated.
+                    Element.SendKeys(Keys.Enter);
+                    WaitForUrlToContain(url);
+                }

```

Comment:
What is the difference between the if & else statement here? They are both doing the same thing

Reply:
Agreed … removing this